src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -97,15 +97,7 @@ trait WorkflowComponent {
     }
     
     def getWithWorkflowIds(workspaceContext: SlickWorkspaceContext, submissionId: String): ReadAction[Seq[(Long, Workflow)]] = {
```

Comment:
This is only used in tests - do we still need it?  If we do, refactor this to return the workflow IDs only, since that's the only piece used by the tests.